src/main/java/uk/gov/pay/api/model/directdebit/agreement/GetAgreementResponse.java

```diff
@@ -5,7 +5,7 @@
 import io.swagger.annotations.ApiModelProperty;
 import uk.gov.pay.api.model.links.directdebit.AgreementLinks;
 
-@ApiModel(value = "CreateAgreementResponse", description = "The Agreement Payload to create a new Agreement")
+@ApiModel(value = "CreateMandateResponse", description = "The Agreement Payload to create a new Agreement")
 public class GetAgreementResponse {
 
     private static final String AGREEMENT_ID_FIELD_NAME = "agreement_id";
```

Comment:
is this still used?

Reply:
i think this should be covered in https://payments-platform.atlassian.net/browse/PP-5299

Reply:
Aaah 👍